toolbox/db/db_set.m

```diff
@@ -21,19 +21,25 @@
 % =============================================================================@
 %
 % Authors: Martin Cousineau, 2020
+%          Raymundo Cassani, 2021
 
 % Parse inputs
-if isempty(varargin{1})
-    sqlConn = sql_connect();
+if isjava(varargin{1}) && nargin > 1
+    handleConn = 0;         
+    sqlConn = varargin{1};  
+    varargin(1) = [];
+end
+if (nargin >= 1) && ischar(varargin{1}) 
+    args = {};
     handleConn = 1;
+    sqlConn = sql_connect();
+    contextName = varargin{1};
+    if nargin > 1
+        args = varargin(2:end);
+    end
 else
-    sqlConn = varargin{1};
-    handleConn = 0;
+    return
```

Comment:
Same comment here, you might want an error message here rather than returning nothing.